pkg/clients/network/network.go

```diff
@@ -19,7 +19,7 @@ package network
 import (
 	"reflect"
 
-	networkmgmt "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2019-06-01/network"
+	networkmgmt "github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-07-01/network"
```

Comment:
@guilhem do you know if there are any behavior changes between the versions?

Reply:
@muvaf I didn't see that much change. Only new resources exposed :)
But to be honest, I can remove this change and bump version only when needed in another PR.

Reply:
Got it. Do you plan on making that new change soon?

Reply:
@muvaf done :)